src/test/groovy/unit/org/pih/warehouse/inventory/CycleCountTransactionServiceSpec.groovy

```diff
@@ -110,15 +107,15 @@ class CycleCountTransactionServiceSpec extends Specification implements DataTest
                 ]
         )
 
-        and: 'a mocked product inventory transaction'
-        createExpectedProductInventoryTransaction(facility, product, cycleCount, date)
+        and: 'a stubbed baseline transaction that will be created for the product'
+        stubExpectedBaselineTransaction(product)
 
         when:
         List<Transaction> transactions = cycleCountTransactionService.createTransactions(cycleCount, [product], true)
 
-        then: 'the only transaction should be the product inventory one'
+        then: 'the only transaction should be the baseline'
         assert transactions.size() == 1
-        assert transactions.findAll{ it.transactionType != adjustmentTransactionType }.size() == 1
+        assert transactions[0].transactionType == baselineTransactionType
```

Comment:
Now the tests actually assert that the baselineTransactionType is returned for the stubbed baseline transaction (before it just asserted that transactionType != adjustmentTransactionType)